src/NuGet.Indexing/ResponseFormatter.cs

```diff
@@ -346,7 +346,19 @@ public static void WriteStatsResult(JsonWriter jsonWriter, NuGetIndexSearcher se
             jsonWriter.WriteStartObject();
             WriteProperty(jsonWriter, "numDocs", searcher.IndexReader.NumDocs());
             WriteProperty(jsonWriter, "indexName", searcher.Manager.IndexName);
+            WriteProperty(jsonWriter, "machineName", searcher.MachineName);
             WriteProperty(jsonWriter, "lastReopen", searcher.LastReopen);
+            WriteProperty(jsonWriter, "lastIndexReloadTime", searcher.LastIndexReloadTime);
+            WriteProperty(jsonWriter, "lastIndexReloadDurationInMilliseconds", searcher.LastIndexReloadDurationInMilliseconds);
```

Comment:
Currently, we don't have the service itself reporting any metrics. We can either have the service itself report these counters to geneva, or have this endpoint report it and have the runner report to geneva.

We current have the structure for the latter, and i don't think we should split it up and report some metrics directly from the service and others from the runner. Since the runner is already there, I would like to leverage that, but if we want to move to letting each instance report directly, that's a change we should discuss with the wider team (ie should all our services self report).